nimare/dataset.py

```diff
@@ -638,11 +642,11 @@ def get_studies_by_mask(self, mask):
 
         dset_mask = self.masker.mask_img
         if not np.array_equal(dset_mask.affine, mask.affine):
-            from nilearn.image import resample_to_img
+            LGR.warning("Mask affine does not match Dataset affine. Assuming same space.")
 
-            mask = resample_to_img(mask, dset_mask, interpolation="nearest")
+        dset_ijk = mm2vox(self.coordinates[["x", "y", "z"]].values, mask.affine)
```

Comment:
`resample_to_img` assumes the images are already aligned (meaning we never had a method for using masks in different spaces), so I think this makes sense.